integration/cursors/tailable_test.go

```
@@ -41,11 +44,299 @@ func TestCursorsTailable(t *testing.T) {
 				Code: 2,
 				Name: "BadValue",
 				Message: "error processing query: " +
-					"ns=TestCursorsTailable-NonCapped.TestCursorsTailable-NonCappedTree: $and\nSort: {}\nProj: {}\n " +
+					"ns=TestCursorsTailableErrors-NonCapped.TestCursorsTailableErrors-NonCappedTree: $and\nSort: {}\nProj: {}\n " +
 					"tailable cursor requested on non capped collection",
 			}
 			integration.AssertEqualAltCommandError(t, expected, "tailable cursor requested on non capped collection", err)
 			assert.Nil(t, cursor)
 		}
 	})
+
+	t.Run("GetMoreDifferentCollection", func(t *testing.T) {
+		t.Parallel()
+
+		if !setup.IsMongoDB(t) {
+			t.Skip("https://github.com/FerretDB/FerretDB/issues/2283")
```

Let's use expected failure instead of just skipping